modules/client/src/client-common/interfaces/context.ts

```diff
@@ -30,19 +35,22 @@ export type ContextParams = Web3ContextParams &
 // Context state data
 
 type Web3ContextState = {
-  network: Networkish;
-  signer?: Signer;
-  daoFactoryAddress?: string;
-  daoRegistryAddress?: string;
-  pluginRepoRegistryAddress?: string;
+  network: Network;
```

Comment:
What's the difference between Networkish and Network?

I wonder wether we want to couple things with Ethers. It might be inevitable

Reply:
Networkish is basically `string | number | Network`, `Network` is the object used by ethers the structure is teh following:

```
{
    chainId: number,
    name: string,
    ensAddress?: string,
}
```
So the idea is that we receive a `Networkish`, which can be a network, a string with the name or the chain id, and we change it to a `Netowrk` type so it can be usable by the `JsonRPCProvider`